CMakeLists.txt

```diff
@@ -81,7 +81,7 @@ if (WIN32)
 endif (WIN32)
 
 # Define the main library.
-add_library(snappy SHARED
+add_library(Snappy SHARED
```

Comment:
https://github.com/Optiligence/snappy/commit/fb66b509cdb31687414654f158ac4617d18bc1af#diff-af3b638bc2a3e6c650974192a53c7291L96
The output name stays the same. 
CMake targest are usuaply upper cased.